### ISSUE TITLE
Cannot open files for UI

### ISSUE THREAD

**Issue by gcolajan** [radare/radare2#7199]
Steps to reproduce:

- Clone radare2-webui
- Build one `make material`
- Run it: `make runm`
- Go to the UI: http://localhost:9090/m/

Current output:
```
Starting http server...
open http://localhost:9090/
r2 -C http://localhost:9090/cmd/
[HTTP] 127.0.0.1:2135 /m
[HTTP] 127.0.0.1:2136 /m/
http: Cannot open '<MY_PATH>/radare2-webui/dev//m/index.html'
[HTTP] 127.0.0.1:2137 /favicon.ico
```

The path indicated is correct. I can reach it and has proper rights.

If `-e http.sandbox=false` is added to original command: `r2 -q -e http.homeroot=dev -e http.ui=m -c=H /bin/ls`, everything works as expected.

**Comment by radare** [comment]
I think using absolute paths for homewww and such should work. Plz confirm

**Comment by gcolajan** [comment]
Yes, it's working: `r2 -q -e http.homeroot=<MY_PATH>/radare2-webui/dev/m  -c=H /bin/ls`

**Comment by radare** [comment]
we can fix that temporarly by just using $(shell pwd) in that line of the makefile. but i think we should avoid relative paths in those http vars

**Comment by radare** [comment]
Then the solution would be to abspath before using those paths in r2. I will have a look at this later this week

**Comment by radare** [comment]
all the webuis are broken, cant update at all for about 3 releases. so not gonna fix this until the webuis work again

**Comment by radare** [comment]
can you try again?

**Comment by XVilka** [comment]
It is a loooong try.

**Comment by novia713** [comment]
this thing is happening to me also :-(

**Comment by radare** [comment]
disable the http.sandbox
